ush/forecast_predet.sh

```diff
@@ -52,6 +52,7 @@ common_predet(){
   CDUMP=${CDUMP:-gdas}
   CASE=${CASE:-C768}
   CDATE=${CDATE:-2017032500}
+  ENSMEM=${ENSMEM:-000}
```

Comment:
I think this should be blank.
I need to inspect what is being doing in `forecast_postdet.sh` and raise the question why is this even relevant in the GFS application.  This seems like an remnant from P8 development where the output was hard-wired.
```suggestion
  ENSMEM=${ENSMEM:-000}
```
```suggestion
  ENSMEM=${ENSMEM:-}
```